UnitTests/expected/Storyboards-Message-Lowercase.swift.out

```diff
@@ -26,47 +26,45 @@ extension StoryboardScene where Self: RawRepresentable, Self.RawValue == String
   }
 }
 
-protocol StoryboardSegue : RawRepresentable { }
+protocol StoryboardSegueType : RawRepresentable { }
 
 extension UIViewController {
-  func performSegue<S : StoryboardSegue where S.RawValue == String>(segue: S, sender: AnyObject? = nil) {
+  func performSegue<S : StoryboardSegueType where S.RawValue == String>(segue: S, sender: AnyObject? = nil) {
     performSegueWithIdentifier(segue.rawValue, sender: sender)
   }
 }
 
-extension UIStoryboard {
-  struct Scene {
-    enum Message : String, StoryboardScene {
-      static let storyboardName = "Message"
+struct Scene {
```

Comment:
Replace with StoryboardScene (here and in other places; same for Segue of course)